simplecpp.cpp

```diff
@@ -1002,7 +1002,8 @@ void simplecpp::TokenList::combineOperators()
                 continue;
             }
             // float literals..
-            if (tok->previous && tok->previous->number && sameline(tok->previous, tok)) {
+            if (tok->previous && tok->previous->number && sameline(tok->previous, tok) && tok->previous->str().find('_', 0) == std::string::npos
+                && tok->previous->str().find('.', 0) == std::string::npos) {
```

Comment:
hmm `tok->previous->str().find('_', 0) == std::string::npos && tok->previous->str().find('.', 0) == std::string::npos` can be simplified to `tok->previous->str().find_first_of("._") == std::string::npos`